test/integration/simple-deployment-evaluation/00-install.yaml

```diff
@@ -0,0 +1,63 @@
+
+
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnEvaluationDefinition
+metadata:
+  name: pre-deployment-hello
+spec:
+  objectives:
+    - evaluationTarget: <1
```

Comment:
i would change this target to something like `>0.1` - otherwise chances are high that this test will not pass on environments with more CPU resources

Reply:
the value we get in this test is -10 or something, so that won't be sufficient

Reply:
We should then change the test, not the example. I suggest rewriting the query with just `sum(kube_node_status_capacity{resource='cpu'})`

Reply:
then the query seems to be wrong - can you maybe try with the following?

```
sum(kube_node_status_capacity{resource='cpu'}) - sum(kube_pod_container_resource_requests{resource='cpu'}) 
```

Reply:
@thisthat 's suggestion might be the more robust option

Reply:
we have this value all over examples, follow up or this PR?

Reply:
a follow up PR would be OK in my opinion, but I would suggest to revert the change of the evaluationTarget in the example (see comment above)

Reply:
fixed